### DISCUSSION TITLE
What's the update route from 2.x to 3.00?

### DISCUSSION THREAD

**Discussion by elliot-100** [Devolutions/UniGetUI#1961] [Q&A] [answered]
I'm on 2.2.0 and the auto-updater doesn't seem to be updating to 3.00 - should it?

If I run the 3.00 installer, it doesn't seem to mention an existing installation.

Thanks!

**Comment by marticliment** [comment, marked as answer]
> I'm on 2.2.0 and the auto-updater doesn't seem to be updating to 3.00 - should it?

It should, can you please share your WingetUI Version 2.2.0 logs?

**Reply by elliot-100** [reply]
Sure, here's the log:

[wingetui_log.txt](https://github.com/marticliment/WingetUI/files/14618816/wingetui_log.txt)

I guess this bit is significant?

```
🟢 Updates found!
🟠 Hash not ok
```

Thanks for this and for the app!

**Reply by marticliment** [reply]
Can you please post here what text do you see if you open the following file in a browser:
https://www.marticliment.com/versions/wingetui.ver

**Reply by elliot-100** [reply]
3.01///dabf9a8115e8cf4f3dab56a61b69d354fbfeb5eaacdd50e96be9c2597b0691fe

However, today it has updated itself to 3.01 OK 👍
